fastlane/Fastfile

```diff
@@ -69,14 +69,22 @@ lane :release do
         skip_upload_screenshots: true
     )
 
-
     sentry_create_release(
         org_slug: 'openfoodfacts',
         project_slug: 'openfoodfacts-android',
         version: PACKAGE_NAME + '@' + versionName + '+' + versionCode,
         finalize: false # Release will be finalized by the "finalize_sentry" lane, when the version is in production
     )
 
+    FLAVOR = SharedValues::GRADLE_FLAVOR + SharedValues::GRADLE_BUILD_TYPE
+    sentry_upload_proguard(
+      auth_token: ENV['SENTRY_AUTH_TOKEN'],
+      org_slug: 'openfoodfacts',
+      project_slug: 'openfoodfacts-android',
+      android_manifest_path: './app/build/intermediates/merged_manifests/' + FLAVOR + '/out/AndroidManifest.xml'
+      mapping_path: './app/build/outputs/mapping/' + FLAVOR + '/mapping.txt',
+    )
+
```

Comment:
could this be set before `sentry_create_release`? Doing so we keep the same order of execution as before, when the upload was part of the build. It also means no sentry release will be created if we cannot upload the mappings, which is a good idea.

Reply:
ah I thought that `sentry_create_release` is build the apk, now I see that it was done before by pure gradle task
so yes, will move it before `sentry_create_release`